server/models/schema.sql

```diff
@@ -778,6 +778,8 @@ CREATE TABLE `inventory` (
   `last_purchase` DATE NULL COMMENT 'This element allows to store the date of the last purchase order of the product in order to allow the calculation without making much of the average ordering INTerval',
   `num_purchase` INT(10) UNSIGNED NOT NULL DEFAULT 0 COMMENT 'Number of purchase orders' ,
   `num_delivery` INT(10) UNSIGNED NOT NULL DEFAULT 0 COMMENT 'Number of stock delivery' ,
+  `tracker` TINYINT(1) NOT NULL DEFAULT 0 ,
+  `importance` VARCHAR(20) NULL COMMENT 'Inventory level of importance : LOW, MID, HIGH' ,
```

Comment:
Can we make these integers?  That way, we can use mathematical operators for things like "Anything above mid importance".

Right now if we want to show "mid and higher", we have to do ` if (importance === "MID" || importance ===  "HIGH")`.  But it might be easier to say `if (importance > 2) { // do something }`.

Reply:
You would then need to add this to the bhConstants like our Flux IDs.